### PR TITLE
Perfecto upload 0.0.1

### PR DESCRIPTION
**Pull request by sreevatsa-ramya**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by Szabadember** [comment]
Hi @sreevatsa-ramya 
your step.yml seems invalid, please check our contribution guidelines.

`Invalid spec
/steps/perfecto-upload/info: {} "maintainer" value is required`

File: steps/perfecto-upload/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+support_url: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload/issues
+published_at: 2020-06-12T18:25:05.619379+05:30
+source:
+  git: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload.git
+  commit: fa36cef919a0dd841a91e87cd86cce1c3d39ff2a
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- android
```
Comment (koral--) [inline steps/perfecto-upload/0.0.1/step.yml:17]: Flutter should be also there

File: steps/perfecto-upload/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    is_required: true
+    is_sensitive: true
+    summary: Your perfecto access key.
+    title: Perfecto access key
+  perfecto_accesskey: $PERFECTO_ACCESS_KEY
+- opts:
+    description: |
+      The perfect cloud name
+    is_required: true
+    is_sensitive: true
```
Comment (koral--) [inline steps/perfecto-upload/0.0.1/step.yml:63]: Why is this sensitive?

File: steps/perfecto-upload/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    summary: Your perfecto cloud name. Ex - abc.perfectomobile.com
+    title: Perfecto cloud name
+  perfecto_cloudname: $PERFECTO_CLOUD_NAME
+- opts:
+    description: |
+      File path on perfecto cloud repository where it would be stored
+    is_required: true
+    is_sensitive: true
```
Comment (koral--) [inline steps/perfecto-upload/0.0.1/step.yml:71]: Why is this sensitive?

File: steps/perfecto-upload/0.0.1/step.yml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  - name: jq
+  - name: curl
+  apt_get:
+  - name: git
```
Comment (koral--) [inline steps/perfecto-upload/0.0.1/step.yml:34]: Only curl seems to be used in script.

File: steps/perfecto-upload/0.0.1/step.yml (hunk 2 above)
Comment (sreevatsa-ramya) [reply steps/perfecto-upload/0.0.1/step.yml:63]: I have removed it

File: steps/perfecto-upload/0.0.1/step.yml (hunk 1 above)
Comment (sreevatsa-ramya) [reply steps/perfecto-upload/0.0.1/step.yml:17]: Have included it

File: steps/perfecto-upload/0.0.1/step.yml (hunk 4 above)
Comment (sreevatsa-ramya) [reply steps/perfecto-upload/0.0.1/step.yml:34]: Have removed the unused ones

**Comment by sreevatsa-ramya** [comment]
Hi,

Please let me know where should I specify the "maintainer" value in
step.yml. I checked step guidelines and also multiple steps in the
bitrise repo but I couldn't find it anywhere. Please advise

BR,
Sreevatsa

File: steps/perfecto-upload/0.0.1/step.yml [hunk 5]
```diff
@@ -0,0 +1,74 @@
+title: perfecto-upload
```
Comment (trapacska) [inline steps/perfecto-upload/0.0.1/step.yml:1]: Please use a more UX friendly name here, for example `Perfecto Upload`. This is the name the users will see in the workflow editor step selector.

Comment (sreevatsa-ramya) [reply steps/perfecto-upload/0.0.1/step.yml:1]: Thanks. Have done

**Comment by sreevatsa-ramya** [comment]
Hi,

I have done the changes and have shared the step again.

Thanks,
Sree

**Comment by koral--** [comment]
There should be one commit for PR (except optional one for icon).

**Pull request closed without merging**